spec/domain/helpers/dates_spec.exs

```diff
@@ -0,0 +1,82 @@
+defmodule Squadster.Domain.Helpers.DatesSpec do
+  use ESpec.Phoenix, async: true
+
+  alias Squadster.Helpers.Dates
+
+  let :date_string, do: "21.07.2000"
+  let :date, do: ~D[2000-07-21]
+
+  let :datetime_string, do: "21.07.2000 12:15"
+  let :datetime, do: ~U[2000-07-21 12:15:03.15Z]
+  let :datetime_without_microseconds, do: ~U[2000-07-21 12:15:03Z]
+
+  let :naive_datetime, do: ~N[2000-07-21 12:15:00]
+
+  describe "date_from_string/1" do
+    it "parses Date from string with '%-d.%-m.%Y' format" do
+      expect Dates.date_from_string(date_string()) |> to(eq date())
+    end
+
+    context "when one-digit date elements passed" do
```

Comment:
I think, that we don't need to test it.

Reply:
As far as it was the bug case, I think it should be tested